rita/src/rita_exit/db_client/mod.rs

```diff
@@ -89,6 +89,7 @@ fn get_country(ip: &IpAddr) -> Result<String, Error> {
 }
 
 #[test]
+#[ignore]
```

Comment:
is this sneaking in from another change?

Reply:
I was playing with the tests and this broke on me because my internet was a bit flaky, which made me realize that we shouldn't run this by default